Makefile

```diff
@@ -14,7 +14,13 @@ logs: up
 
 dependencies:
 	pip install --upgrade pip
+	pip install -r requirements.txt
 	pip install -r tests/requirements.txt
 
-test: dependencies up
+test: functional static_analysis
+
+functional: dependencies up
 	pytest --noconftest tests/functional
+
+static_analysis: dependencies
```

Comment:
Could you add how to run this in the read me please?

Reply:
Done